tests/test_monkeypatch.py

```diff
@@ -33,3 +36,20 @@ def test_view_published_in_toolbar_in_preview_mode_button_url(self):
         self.assertContains(
             response,  'class="cms-btn cms-btn cms-btn-switch-save" target="_blank" >View Published</a>'
         )
+
+
+class VersioningIntegrationTestCase(CMSTestCase):
+
+    def test_versioning_changelist_published_date(self):
+        """
+        Monkey patch should add expiry column and values to admin menu list display
+        """
+        published_version = factories.PageVersionFactory(content__template="page.html", content__language="en")
+        version_admin = admin.site._registry[published_version.versionable.version_model_proxy]
+
+        request = RequestFactory().get("/")
+        list_display = version_admin.get_list_display(request)
+
+        # Published date field should have been added by the monkeypatch
+        self.assertIn('published_date', list_display)
```

Comment:
```suggestion
        self.assertIn('published_date', list_display)
        self.assertNotIn('created', list_display)
```
Could be worth adding as assert to check that the `created` field has been removed